Nos/Models/Profile.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Profile.swift
+//  Nos
+//
+//  Created by Martin Dutra on 27/4/23.
+//
+
+import Foundation
+import secp256k1
+
+/// A struct that abstracts the information included ina nprofile string.
+///
+/// See https://github.com/nostr-protocol/nips/blob/master/19.md
+/// In the future we can parse and include the relays in this struct, but now we just need the public key.
+struct Profile {
```

Comment:
Maybe best to call this NProfile since we already have things like the `ProfileView` that aren't related to this model.